testpb/1.pulsar.go

```diff
@@ -1,14 +1,22 @@
 package testpb
 
 import (
+	binary "encoding/binary"
 	fmt "fmt"
 	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
 	protoiface "google.golang.org/protobuf/runtime/protoiface"
 	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	io "io"
+	math "math"
+	bits "math/bits"
 	reflect "reflect"
 	sync "sync"
 )
 
+func (x *fastReflection_A) Unwrap() *A {
+	return (*A)(x)
```

Comment:
I'm not understanding the purpose of this method, since `*A` is a concrete type and not an interface type, and access to `fastReflection_A`, which is private, is given only through `*A.ProtoReflect()` which returns an interface type.